docs/specs/git.yml

```diff
@@ -334,24 +334,6 @@ outputs:
       "updated_at": "2018-10-30 12:00 AM",
     }
 ---
-# Use commit build time as updated_at when `updateTimeAsCommitBuildTime` is true and fallback to old style build_history
-repos:
-  https://docs.com/update_time_as_commit_build_time#master:
-    - files:
-        docfx.yml: |
-          updateTimeAsCommitBuildTime: true
-        docs/index.md: Alice creates this.
-state:
-  history/build_history_1026df13-852a-b3da-a226-492390c1c82e_17190aeb-6297-d34d-a48f-a681d3061212.json: |
```

Comment:
Instead of removing the test case, is updating this to `build-history.json` work?

Reply:
updated and searched both build-history and build_history in test cases. No other findings.